checkfiles.py

```diff
@@ -1002,11 +1002,16 @@ def fetch_files(session, url, search_query, out, include_unexpired_upload=False,
         }
         errors = job['errors']
         item_url = urljoin(url, job['@id'])
-
+        fileObject = session.get(item_url)
```

Comment:
I think you need to handle HTTP errors here. Probably want to add to the `job["errors"]`, rather than the usual `response.raise_for_status()` so the code can continue. If you get an error then you shouldn't attempt to get the `s3_uri`.
```suggestion
        fileObject = session.get(item_url)
        if not fileObject.ok:
            # handle the error
```